Library/include/CSP/Systems/Organizations/OrganizationSystem.h

```diff
@@ -53,6 +53,49 @@ class CSP_API CSP_NO_DISPOSE OrganizationSystem : public SystemBase
 	/// @param Callback MemberJoinedOrganizationCallback : the callback to execute.
 	CSP_EVENT void SetMemberJoinedOrganizationCallback(MemberJoinedOrganizationCallback Callback);
 
+	/// @brief Retrieves Organization info for the specified Organization.
+	/// If this request is made by a User with an Owner or Admin Organization role, the resultant Organization object will contain an array of
+	/// OrganizationRoleInfo objects for each Organization member. If the request is made by a User who does not have the Owner or Admin role, the
+	/// resultant Organization object will contain an array with a single OrganizationRoleInfo object which represents them.
+	/// @param OrganizationId csp::common::Optional<csp::common::String> : Id of the Organization to retrieve information on. If no Id is specified,
+	/// the Id of the Organization the user is currently authenticated against will be used.
+	/// @param Callback OrganizationResultCallback : Callback when asynchronous task finishes.
+	CSP_ASYNC_RESULT void GetOrganization(const csp::common::Optional<csp::common::String>& OrganizationId, OrganizationResultCallback Callback);
```

Comment:
nitpick - feel free to mark resolved if you don't want it - but I prefer CRUD APIs to start with Create, as that's the first thing a developer will need to do when starting their journey adding support for Organizations.

Reply:
Agreed I have moved the CreateOrganization method.